p/scripts/main.js

```diff
@@ -1546,11 +1546,12 @@ function load_more_posts() {
 		formPagination.replaceChild(paginationNew, paginationOld);
 
 		const bigMarkAsRead = document.getElementById('bigMarkAsRead');
-		if (bigMarkAsRead) {
+		const readAll = document.querySelector('#nav_menu_read_all .read_all');
+		if (bigMarkAsRead.tagName.toLowerCase == 'button' && readAll) {
```

Comment:
```suggestion
		if (readAll && bigMarkAsRead && bigMarkAsRead.formAction) {
```

Sorry for the delay. I did not like that line so much, and had to look more closely at it. Could you please try this suggestion?

Reply:
Looks fine too